### PR TITLE
Style guide | Small loader

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #2795

## AC
- [x] Add a section titled "Small Loader" between Loader and Loader buttons 
- [x] Loading default message on style guide should say **_Loading..._** in font-weight 700, italicized and in 1.9 rem. 
- [x] Use spinner from Loader button and change size to be 19 x 19px

smallloader

### CODE REVIEW

File: client/app/containers/StyleGuide/StyleGuideSmallLoader.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        title="Small Loader"
+        id="small_loader"
+        link="StyleGuideSmallLoader.jsx"
+      />
+      <p>
+      The small loading indicator can be used when information is still being pulled
+      for a specific section on a page. Designers are responsible to indicate what the
+      message should say and which application logo to use in the acceptance criteria. </p>
+
+      <div className="cf-sg-loader-font">
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideSmallLoader.jsx]: You should make this into a React component

File: db/schema.rb [hunk 2]
```diff
@@ -12,6 +12,7 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 20170815120508) do
+
```
Comment (mmlumba) [inline db/schema.rb]: This shouldn't be updated

Comment (Sjones352) [inline db/schema.rb]: Never updated this

Comment (mmlumba) [inline db/schema.rb]: You can remove this new line so that way Git won't check this updated version into version control

**Pull request merged**